### PR TITLE
Fill in Hello World section

### PR DESCRIPTION
**Pull request by brooksdavis**
Also some improvements to packages wording.

My examples are all in a (modified to be larger) Qemu image. We may want to redo them with a proper build so they have the right hostname and paths.

### CODE REVIEW

File: src/helloworld/README.md [hunk 1]
```diff
@@ -3,27 +3,145 @@
 These instructions are intended for use on an Arm Morello board, and install
 hybrid ABI versions of key toolchain and utilities using `pkg64`.
```
Comment (rwatson) [inline src/helloworld/README.md:4]: I guess we should make some sort of apologetic noise about CHERI-RISC-V, with the promise that that will follow in due course?

File: src/helloworld/README.md [hunk 2]
```diff
@@ -3,27 +3,145 @@
...
+...
+[11/11] Installing llvm-13,1...
+[11/11] Extracting llvm-13,1: 100%
+=====
+...
+```
+
+*Note:* By default FreeBSD ships with the `vi` and `ee` editors. You may
+wish to install the `emacs-nox`, `nano`, or `vim` package to access a
+more familiar editor.
```
Comment (rwatson) [inline src/helloworld/README.md:64]: Do we want to hint here that `pkg64` should be used, as above?

File: src/helloworld/README.md [hunk 3]
```diff
@@ -3,27 +3,145 @@
...
+*Note:* Currently, clang defaults to targeting standard Armv8-a rather
+than Morello so you must add `-march=morello` to use CHERI extensions
+and `-mabi=purecap` to build CheriABI binaries.
+
 ## Running
 
+Run the program:
+
+```
+root@cheribsd-morello-purecap:~ # ./helloworld
```
Comment (rwatson) [inline src/helloworld/README.md:93]: Most Morello board installs will use the hostname `cheribsd` now.

File: src/helloworld/README.md [hunk 4]
```diff
@@ -3,27 +3,145 @@
...
-
 	printf("Hello world\n");
 }
 ```
 
 ## Building
 
+To build a CheriABI Hello World program use:
+
+```clang -g -o helloworld -march=morello -mabi=purecap helloworld.c```
```
Comment (rwatson) [inline src/helloworld/README.md:82]: I wonder if we should encourage using `-Wall` here, leading by example..?

File: src/packages/README.md [hunk 5]
```diff
@@ -9,14 +9,28 @@ They have different levels of completeness, maturity, security, and support.
...
+  `/usr/local64` for dependencies. Please report bugs of this sort in
+  the [CheriBSD ports issue tracker](https://github.com/CTSRD-CHERI/cheribsd-ports/issues).
+
 - **CheriABI packages** are compiled using pure-capability CHERI C/C++, and
   employ fine-grained C/C++ memory protection.
-  There are in the range of 1,000-3,000 CheriABI packages available.
-  These are considered *experimental*.
-  command.
+  There are currently over 8,000 CheriABI packages available.
+  These packaages are considered *experimental*.
```
Comment (rwatson) [inline src/packages/README.md:26]: s/packaages/packages/

File: src/helloworld/README.md (hunk 1 above)
Comment (brooksdavis) [reply src/helloworld/README.md:4]: Yeah, I'll add some words here and in packages.

**Review by rwatson — APPROVED** [review]

**Pull request merged**